pycg/machinery/definitions.py

```diff
@@ -192,10 +194,11 @@ class Definition(object):
         utils.constants.EXT_DEF,
     ]
 
-    def __init__(self, fullns, def_type):
+    def __init__(self, fullns, def_type, lineno = None):
         self.fullns = fullns
         self.points_to = {"lit": LiteralPointer(), "name": NamePointer()}
         self.def_type = def_type
+        self.lineno = lineno
```

Comment:
This should be able to hold more than just lineno. Therefore it should be a dict called `self.metadata`. and the dict can hold for now, "lineno" and "col_offset"

Reply:
And as discussed, there should be a list of lineno and col_offset rather than just one per definitions. You can call this list `self.defined_at` 

e.g., `self.defined_at = [{"lineno": 1, "col_offset": 1}, {"lineno": 2, "col_offset": 1}]`